.github/ISSUE_TEMPLATE/bug_report_form.yml

```diff
@@ -36,6 +36,12 @@ body:
       label: Vue version
       description: |
         See package.json
+  - type: input
+    id: library-version
+    attributes:
+      label: Library version
+      description: |
+        See apps admin page, e.g. 0.5.3
```

Comment:
omit the description. reporters should know what version they run and where to find that

for a library that is not found anywhere on the UI